tedbottompicker/build.gradle

```diff
@@ -61,7 +61,7 @@ dependencies {
     compile 'com.android.support:support-annotations:25.1.0'
     javadocDeps 'com.android.support:support-annotations:25.1.0'
     compile 'com.github.bumptech.glide:glide:3.7.0'
-    compile 'com.tbruyelle.rxpermissions:rxpermissions:0.9.1@aar'
+    compile 'com.tbruyelle.rxpermissions:rxpermissions:0.7.0@aar'
```

Comment:
we still use this version in our app => we can upgrade here if we upgrade ourselves